services/debugger/uploader.go

```diff
@@ -111,10 +115,11 @@ func (uploader *uploaderImpl[E]) uploadEvents(eventBuffer []E) {
 	url := uploader.url
 
 	retryCount := 1
+	defer uploader.timingStat.RecordDuration()()
```

Comment:
Is it useful to include potential retries in this measurement? It might be better to use this timer only for measuring a single request & add another counter for capturing errors too.

Btw, since a timer includes a counter, we could limit our stats to the following 2:
1. Timer `debugger_http_requests` with tags `responseCode` & `endpoint` for capturing successful requests
2. Counter `debugger_http_errors` with tags `endpoint` for capturing failed requests (with `err`)